src/main/java/com/checkmarx/ast/wrapper/CxWrapper.java

```diff
@@ -274,7 +278,10 @@ public String results(@NonNull UUID scanId, ReportFormat reportFormat)
         arguments.add(fileName);
         arguments.add(CxConstants.OUTPUT_PATH);
         arguments.add(tempDir);
```

Comment:
Let's talk about the PR. Im thinking to have a default call for wrappers, as Plugins maybe